ContextApp/src/main/java/com/github/migi_1/ContextApp/HelloActivity.java

```diff
@@ -21,11 +31,82 @@ public HelloActivity(){
 //        // Enable verbose logging
 ////        eglConfigVerboseLogging = false;
 //        // Choose screen orientation
-////        screenOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE;
-//        // Enable MouseEvents being generated from TouchEvents (default = true)
+          // Enable MouseEvents being generated from TouchEvents (default = true)
 //        mouseEventsEnabled = true;
 //        // Set the default logging level (default=Level.INFO, Level.ALL=All Debug Info)
-//        LogManager.getLogManager().getLogger("").setLevel(Level.INFO);
+        LogManager.getLogManager().getLogger("").setLevel(Level.INFO);
+        
+    }
+
+         @Override  
+    protected void onResume()  
+    {  
```

Comment:
Be consistent with the method brackets please.

method() {
}

Instead of

method()
{
}